osbs/repo_utils.py

```diff
@@ -82,13 +132,12 @@ def __init__(self, dir_path='', file_name=REPO_CONFIG_FILE, depth=None,
 
         file_path = os.path.join(dir_path, REPO_CONTAINER_CONFIG)
         if os.path.exists(file_path):
-            with open(file_path) as f:
-                try:
-                    self.container = yaml.safe_load(f) or {}
-                except yaml.scanner.ScannerError as e:
-                    msg = ('Failed to parse YAML file "{file}": {reason}'
-                           .format(file=REPO_CONTAINER_CONFIG, reason=e))
-                    raise OsbsException(msg)
+            try:
+                self.container = read_yaml_from_file_path(file_path, 'schemas/container.json') or {}
+            except yaml.scanner.ScannerError as e:
```

Comment:
except Exception as e:

msg => Failed to load and validate ....

Reply:
Why go from the specific error type to the general?  The linter seems to prefer specific errors.

Regardless, I improved the error message.

Reply:
read_yaml will return multiple different exceptions, so we want to wrap those in here with OsbsException

Reply:
Okay then, fixed.